packages/openapi-framework/index.ts

```diff
@@ -76,11 +76,21 @@ export default class OpenAPIFramework implements IOpenAPIFramework {
   private securityHandlers;
   private validateApiDoc;
   private validator;
+  private logger;
 
   constructor(protected args = {} as OpenAPIFrameworkConstructorArgs) {
     this.name = args.name;
     this.featureType = args.featureType;
     this.loggingPrefix = args.name ? `${this.name}: ` : '';
+    this.logger = args.logger ? args.logger : console;
+    // monkey patch for node v6:
+    if (!this.logger.debug) {
+      this.logger.debug = this.logger.log;
+    }
+    this.logger.debug(
+      `${this.loggingPrefix}constructing with`,
+      JSON.stringify(args)
```

Comment:
not sure how crazy i am about the `JSON.stringify(args)` here.  can you remove it?

Reply:
i'd actually rather not add this line altogether

Reply:
functions no but all other objects would be nicely serialized. it's much easier to catch an error with enough debugging info printed, especially in production where it's not that easy to debug. I've removed it, perhaps there's a better way.